#! /bin/sh /usr/share/dpatch/dpatch-run
## 10smoker.dpatch by Florian Ragwitz <rafl@debianforum.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust smoker file for build

@DPATCH@

dirr -Nur pugs-6.2.9.orig pugs-6.2.9
--- pugs/util/yaml_harness.pl	(revision 6794)
+++ pugs/util/yaml_harness.pl	(local)
@@ -22,7 +22,7 @@
 use Test::TAP::Model;
 use File::Spec;
 our @ISA = qw(Test::TAP::Model);
-our $SMOKERFILE = ".smoker.yml";
+our $SMOKERFILE = "debian/smoker.yml";
 
 $| = 1;
 
